README.adoc

```diff
@@ -66,9 +66,9 @@ spec:
       nsTemplateSet:
         tierName: basic
         namespaces:
-        - type: ide
+        - type: dev
           revision: abcdef
-        - type: cicd
+        - type: code
```

Comment:
could we also add the `stage` type so we are consistent?

Reply:
done https://github.com/codeready-toolchain/toolchain-common/pull/39/commits/04cbf99bdace9650fe4b73d43537c66e5df9cd58